membership.md

```diff
@@ -0,0 +1,91 @@
+Membership and Commit Rights
+============================
+
+This document covers processes to be granted membership in the Dask
+GitHub organization and commit rights to Dask repositories.
+
+
+Owners
+------
+
+There is a set of individuals who have ownership privileges over the entire
+GitHub organization.  As a result, they have commit rights to all repositories
+and are involved in creating and removing members.  These individuals are
+listed at https://github.com/orgs/dask/people .
+
+We ask that owners ensure that their membership is public.
+
+New owners are made by lazy consensus of previous owners.  All owners will be
+made aware of a new candidate.  Any -1 vote bars a candidate.  We ask that a
+majority of owners respond for quorum, and that those votes come from people
+employed/associated to at least three institutions.
+
+
+Members
+-------
+
+New members can be made by any owner if the following criteria are met:
+
+1.  At least two owners from separate employers agree membership is a good idea
+2.  The new member has supported the project several times, either through code
+    or otherwise, and has demonstrated a willingness to help solve other
+    people's problems, not just their own.
+3.  The new member demonstrates some degree of familiarity of open source
+    practices and online courtesy.
+
+Write privileges
+----------------
+
+Individuals can be given write privileges to any repository by an owner by the
+above conditions and the following:
+
+1.  The lead maintainer of that repository thinks it's a good idea
+2.  The member has demonstrated technical familiarity with git and GitHub (and
+    so, for example, is unlikely to accidentally mess with history or merge PRs
+    in an unclean state)
+
+In the case of the more critical repositories (dask/dask, dask/distributed)
+then at least three owners should agree to give privileges.
+
+
+Teams
+-----
+
+We acknowledge that write permissions to individual projects may be better handled by teams.  Team membership will be handled similar to write permissions.
+
+
+Removing permissions
+--------------------
+
+People's permissions can be removed in a two-step process:
+
+1.  After a year of inactivity they are sent a private message noting that they
+    have been inactive and asking if they'd still like to maintain their
+    permissions.  No response after a suitable time (two weeks) will default in
+    removal of permissions
+2.  After two years of inactivity their permissions are removed regardless
+
+We intentionally call out that "activity" can mean a great many things
+including but not limited to writing code, documenting, being active on issues,
+supporting users, etc..  Activity should be more than token events to maintain
+permissions
+
+Any owner may temporarily remove permissions of any non-owner if there is some
+pressing reason to do so, such as insecure credentials.  A consensus of owners
```

Comment:
Am trying to better understand how "consensus" is used here. Does it mean lazy consensus, unanimous, majority with some percentage, something else?

Reply:
As elsewhere, we've been using lazy consensus.  Is there a change that you would like us to make here?

Reply:
I think we are already having a [discussion in this thread]( https://github.com/dask/governance/pull/11#pullrequestreview-227868480 ). We can update here and elsewhere once we have reached a conclusion.

Reply:
I've pushed a similar change as was used there (e.g. copying from the membership addition policy above). Please let me know your thoughts.